examples/flow.cpp

```diff
@@ -251,15 +251,31 @@ try
     const PhaseUsage pu = Opm::phaseUsageFromDeck(deck);
     Opm::BlackoilOutputWriter outputWriter(grid, param, eclipseState, pu );
 
+    int numCells = Opm::UgGridHelpers::numCells(grid);
```

Comment:
I wasn't thinking of anything fancy: I just meant to say that if `UnstructoredGrid::global_cell` was required to be always non-null, things would be simpler in many places.